build/css/jsxc.oc.css

```diff
@@ -4527,3 +4527,17 @@ div.jsxc_video {
   cursor: pointer; }
   #header .jsxc_chatIcon:hover {
     opacity: 1; }
+
```

Comment:
This should go to https://github.com/nextcloud/jsxc.nextcloud/blob/master/scss/_oc.scss, because otherwise this change will be overwritten on every build.